src/Holiday/Saxony.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * This software is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 3 as published by the Free Software Foundation
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * @copyright  Copyright (c) 2012 Mayflower GmbH (http://www.mayflower.de)
+ * @license    LGPL v3 (See LICENSE file)
+ */
+namespace Holiday;
+
+class Saxony extends Germany
+{
+    protected function getHolidays($year)
+    {
+        $timezone = $this->timezone;
+
+        $data   = parent::getHolidays($year);
+
+        $data[] = new Holiday("31.10." . $year, "Reformationstag", $timezone);
+
+        $date = new Holiday("24.12." . $year, "Buß- und Bettag", $timezone);
```

Comment:
I think you may simplify this calculation with "23.11" and modify "last Wednesday" according to https://de.wikipedia.org/wiki/Gesetzliche_Feiertage_in_Deutschland#.C3.9Cbersicht_aller_gesetzlichen_Feiertage and https://de.wikipedia.org/wiki/Bu%C3%9F-_und_Bettag